lib/PHPExif/Adapter/Native.php

```diff
@@ -345,4 +358,30 @@ public function mapData(array $source)
 
         return $mappedData;
     }
+
+    /**
+     * Extract GPS coordinates from components array
+     *
+     * @param array $components
+     * @return float
+     */
+    protected function extractGPSCoordinate(array $components)
```

Comment:
Can you unit test this method? Coverage is brought down now, which I want to avoid.